src/corehost/cli/args.cpp

```diff
@@ -53,6 +53,16 @@ void setup_shared_store_paths(const hostpolicy_init_t& init, const pal::string_t
         args->dotnet_shared_store = own_dir;
         append_path(&args->dotnet_shared_store, RUNTIME_STORE_DIRECTORY_NAME);
         append_path(&args->dotnet_shared_store, init.tfm.c_str());
+
+        //Also add an architecture specific path to allow for unzipped developer cli experience
```

Comment:
If we take this approach, the comment at the top of this method should be updated.